src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -1316,14 +1312,13 @@ def SourcesDialog
             elsif Popup.ConfirmAbort(:painless)
               break
             end
-          elsif @script_called_from_another
-            break
-          elsif Popup.ContinueCancelHeadline(
-            # TRANSLATORS: popup header
-            _("Aborting Configuration of Online Repository"),
-            # TRANSLATORS: popup question
-            _("Are you sure you want to abort the configuration?")
-          )
+          elsif @script_called_from_another ||
```

Comment:
Is this correct? Previously the popup was not called when `@script_called_from_another` was true...

Reply:
it is not call even now. || and && is short evaluated:

```ruby
irb(main):002:0> true || puts("fail")
=> true
```